src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -492,6 +564,46 @@ private void markAsRead() {
         }
     }
 
+    private void createNewWatchListMenuItem() {
+        Optional<GuiElement> item = listView.getSelectedItem();
+        if (!item.isPresent()) {
+            return;
+        }
```

Comment:
Wait.. I think the method name is wrong, it should be `createNewWatchListPanel`. Will change that in the next push.

Hmm, actually, it should not fail since if there is no selected item, the menu would be disabled (covered in tests already). Maybe an assert would be better?